PizzaLover's pizzaPad

pizzalover125 opened the pull request:
pizzaPad!!!! The one and only pizzaPad!

qcoral reviewed (changes requested):
Hey - awesome submission, couple things:
- I need the source files for the pcb, could you upload a schematic?
- The case needs to be broken into 3D printable parts
- I need screws in the BOM

pizzalover125 commented:
Hello! I updated the BOM and added the part-by-part step files. However, for the PCB, EasyEDA only has the PCB.json export. The only other option is the PDFs, which I have provided.

qcoral reviewed (approved):
nice job, but the vcc and gnd pins are flipped for your OLED footprint - can you fix that?

qcoral reviewed (changes requested):
meant to request changes whoops

pizzalover125 commented:
wait, do I need to import new footprint, or can I just flip them intentionally wrong (right for you)?

qcoral commented:
The latter should work, make sure to reroute your pcb!

Dongathan-Jong commented:
Hey! Hackpad V2 is on, check it out [here](https://hackpad.hackclub.com/). I will be closing this PR since it is not a part of the V2 submission, you are welcome to resubmit, but we have a different structure this time around so be sure to follow the submission guidelines!

pizzalover125 commented:
What is the different structure?
